components/StarkNet/modules/develop/pages/CLI/commands.adoc

```diff
@@ -98,76 +98,63 @@ The following command sends a transaction to the StarkNet sequencer
 $ starknet invoke --gateway_url https://alpha-mainnet.starknet.io/gateway/ <options>
 ----
 
-[id="starknet_deploy_account"]
-== starknet deploy_account
+
+[id="starknet_call"]
+== starknet call
 
 [source,terminal]
 ----
-starknet deploy_account
-  --wallet <wallet_provider>
-  --account <account_name>
+starknet call
+  --address <contract_address>
+  --abi <contract_abi>
+  --function <function_name>
+  --inputs <arguments>
+  --block_hash <block_hash>
+  --block_number <block_number>
+  --signature <signature_information>
+  --wallet <wallet_name>
+  --nonce <nonce>
 ----
 
-Deploys an account contract, accepts the following arguments:
-
-- `account_name` - the name given to the account, used for managing multiple accounts from the CLI (if not specified, the name
-  `+__default__+` is used.
-- `wallet_provider`* - the path to module which manages the account (responsible for key generation, signing, etc.)
+Calls a StarkNet contract without affecting the state, accepts the following arguments:
 
-[NOTE]
-====
+- `contract_address`* - address of the contract being called
```

Comment:
You should convert the structure of all the commands to be similar to what I did in the syscalls PR.

For two-column lists, use the [horizontal,labelwidth=35] prefix (you can play with the labelwidth)